components/tikv_util/src/quota_limiter.rs

```diff
@@ -31,6 +32,10 @@ pub struct QuotaLimiter {
     read_bandwidth_limiter: Limiter,
     // max delay nano seconds
     max_delay_duration: AtomicU64,
+    // if supports auto tune
+    support_auto_tune: AtomicBool,
```

Comment:
What's kind of auto-tune?

Reply:
It's a generic flag. The quota can be adjusted according to the runtime feedback. By this PR, only cpu time quota will be adjusted.